src/Discord.php

```diff
@@ -4,330 +4,87 @@
 
 namespace Exan\Fenrir;
 
-use Discord\Http\Drivers\React;
+use Discord\Http\Drivers\Guzzle;
```

Comment:
You're using Guzzle now?

Reply:
I probably changed this because of the weird slowdown issue I encountered with React, before figuring out Guzzle doesn't play nicely with the react loop. Ill change back to React when we get Bifrost to a usable state.

Reply:
Oh yeah, almost forgot about that. Makes sense, think we should make a note about this somewhere so we don't forget?

Reply:
Ill add an issue to the react drivers repo